src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (height * base) / 2;
```

Comment:
Consider using constants for magic numbers, in this case, the number 2 used in the area calculation formula could be defined as a constant for better clarity and maintainability.